client/app/dedicatedCloud/dashboard/legacy/index.js

```diff
@@ -0,0 +1,14 @@
+import component from './component';
+import {
```

Comment:
What's the benefit of importing the Module and Component name from a constant file? Because they are only used once.

And also, we already use quite a lot this approach:

```js
import angular from 'angular',

import component from './component';

const moduleName = 'myModuleName';

angular
  .module(moduleName, [])
  .component('myComponent', component);

export default moduleName;
```